CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,70 @@
+# Code of conduct for the Particular GitHub community
+
+This code of conduct applies to all contributors, commenters, and participants in Particular Software's GitHub repositories.
+
+These repositories exist to support focused, professional, and technical collaboration around Particular's platform-including contributions to documentation, samples, source code, issue discussions, and community guidance.
+
+We expect all community members to uphold the standards below, in line with the broader [Code of conduct for the Particular community](https://github.com/Particular/DeveloperEducation/blob/master/policies/community-code-of-conduct.md).
```

Comment:
This is a public repo that's been linked to a private repo. We should remove that.